pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -59,6 +60,7 @@ const (
 
 var (
 	AntreaPolicyLogger *log.Logger
+	logDeduplication   logSafeGuard
 )
```

Comment:
why do these need to be 2 separate global variables? Can't they be grouped in one object, which could itself become part of the NetworkPolicyController struct?